naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/schema/NaptimeResourceUtils.scala

```diff
@@ -158,24 +162,27 @@ object NaptimeResourceUtils extends StrictLogging {
 
     relation.arguments
       .mapValues { value =>
-        InterpolationRegex.replaceAllIn(
-          value, { regexMatch =>
-            val withoutBraces = Option(regexMatch.group("withoutBraces"))
-            val withBraces = Option(regexMatch.group("withBraces"))
-            val variableName = withoutBraces.orElse(withBraces).getOrElse("")
-            val fieldName = variableName.split("/").last
-            Utilities.getValueAtPath(
-              data.parentModel.value,
-              data.parentModel.schema,
-              variableName.split("/")).map {
-              case dataList: DataList => dataList.asScala.mkString(",")
-              case other: Any => other.toString
-            }.getOrElse("")
-          })
+        val interpolatedIds: List[String] = InterpolationRegex.findAllMatchIn(value).flatMap { regexMatch =>
```

Comment:
nit: is the type hint required here?

Reply:
I think it increases readability slightly, so will keep.